charts/akeyless-api-gateway/templates/deployment.yaml

```
@@ -237,6 +237,10 @@ spec:
             - name: RESTRICT_SERVICE_TO_ACCESS_IDS
               value: {{ .Values.akeylessUserAuth.restrictServiceToAccessIds | quote }}
           {{- end }}
+          {{- if and .Values.akeylessUserAuth.blockServiceToAccessIds (len .Values.akeylessUserAuth.blockServiceToAccessIds | lt 0) }}
```

The len test is redundent 
```suggestion
          {{- if .Values.akeylessUserAuth.blockServiceToAccessIds }}
```